internal-functions

```diff
@@ -122,5 +122,12 @@ fn-check-service-acl() {
 
 fn-query-admin-status(){
   HEADER="X-Plotly-Dash-Secret-Key: $1"
-  curl -sk --header "$HEADER" "https://$2/v2/users/$3/" | jq '.is_admin' | grep -q 'true'
+  AUTH_URL="$2"
```

Comment:
Can you add define the parameters via `declare`? It makes it clearer what each is.